### PR TITLE
Add README to project.

### LINKED ISSUE #8: Document deployment/release process in README
The deployment process is as follows:

* The default branch is `next`. All PRs should target this branch.
* When ready to release, create a new branch from `next` named `release/x.y.z`
* Bump the version in `package.json` to `x.y.z`
* Merge the **release branch** to `main`. This will trigger the deployment.
* Merge `main` into `next`
* 🎉

### PR DESCRIPTION
**Pull request by jalcine**
Adds setup instructions for the project.

Fixes #8

### CODE REVIEW

**Review by tantek — APPROVED** [review]
Have not tested the setup instructions, but the rest looks good to me. Would be good to get someone familiar with node to try them out and verify they work.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+4. Commit your changes (`git commit -am 'Add some feature'`)
+5. Push to the branch (`git push origin my-new-feature`)
+6. Create new Pull Request
+
+If you find bugs, have feature requests or questions, please
+[file an issue](https://github.com/microformats/microformats-parser-website-node/issues).
+
+## License
+
+Microformats Parser Website Rust is dedicated to the public domain using Creative Commons -- CC0 1.0 Universal.
```
Comment (aciccarello) [inline README.md]: Looks like this was missed.

```suggestion
Microformats Parser Website Node is dedicated to the public domain using Creative Commons -- CC0 1.0 Universal.
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+http://creativecommons.org/publicdomain/zero/1.0
+
+## Contributors
+- [Jacky Alciné](https://github.com/jalcine) &mdash; https://jacky.wtf/about
```
Comment (aciccarello) [inline README.md]: While I like thanking contributors, I feel like it's hard to keep updated manually. Github's own contributor UI is probably a better tool in this case.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+You can view your running local application at this URL:
+
+```
+http://localhost:9080
+```
+
+## Requirements
+
+- [Node 17](https://nodejs.org/en/blog/release/v17.0.0/)
+- [Yarn](https://yarnpkg.com/cli/install)
```
Comment (aciccarello) [inline README.md:50]: Probably should consider yarn optional here as npm does a good enough job installing packages.

Yarn can be a little confusing, particularly given the drastic changes in version 2. I'm okay with recommending it but wouldn't want that to hold someone back from contributing.

**Review by aciccarello — COMMENTED** [review]
I think it'd be great to have this in the repo. Might benefit from a few adjustments.

Regarding the question if the steps work, I was able to get things running with the corresponding `npm install` & `npm start` commands.

File: README.md (hunk 3 above)
Comment (aimee-gm) [reply README.md:50]: Given that the project currently has a `yarn.lock` file (and yarn commands), using npm will create another lockfile. I think it's fine to mention here while this package uses Yarn. Dropping yarn should be a seperate discussion.

File: README.md (hunk 2 above)
Comment (aimee-gm) [reply README.md]: I agree here, and I'll remove this section. It's going to be painful to maintain and will never reflect the true current state of things.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+yarn install
+yarn run start
+# Or, if you'd like a different port:
+# PORT=5000 yarn run start 
+```
+
+You can view your running local application at this URL:
+
+```
+http://localhost:9080
```
Comment (aimee-gm) [inline README.md]: It runs on port `9000`, not `9080`

**Review by aimee-gm — APPROVED** [review]

**Pull request merged**